packages/backend/src/modules/tracked-txs/TrackedTxsModule.ts

```diff
@@ -91,7 +91,7 @@ export function createTrackedTxsModule(
     const coingeckoClient = providers.coingeckoClient
     const coingeckoQueryService = new CoingeckoQueryService(
       coingeckoClient,
-      logger.tag('trackedTxs'),
+      logger.tag({ tag: 'trackedTxs' }),
```

Comment:
feature: trackedTxs

Reply:
trackedTxs is not a feature, added as a module